app/controllers/bales_controller.rb

```diff
@@ -16,6 +17,14 @@ def show
     render json: bale_by_id
   end
 
+  def update
+    if bale_by_id.update(bale_params)
+      head :ok
+    else
+      render_error(1, bale_by_id.errors)
+    end
+  end
+
   private
 
   def bale_by_id
```

Comment:
What do you think about renaming this method as `bale` ? I think it is shorter and the controller is more clear this way